addons/website_crm/views/website_crm_templates.xml

```diff
@@ -1,87 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-        <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website.contactus_form" customize_show="True">
-            <xpath expr="//div[@id='contactus_section']" position="replace">
-                <div id="contactus_section">
-                    <section class="s_website_form" data-vcss="001" data-snippet="s_website_form">
-                        <div class="container">
-                            <form id="contactus_form" action="/website/form/" method="post" enctype="multipart/form-data" data-model_name="crm.lead" data-success-mode="redirect" data-success-page="/contactus-thank-you" class="o_mark_required" data-mark="*" data-editable-form="false">
```

Comment:
Without this, lead are not generated anymore when form is submitted, isn't it?
That's the whole purpose of the module, unless I miss something obvious here it seems strange?

Reply:
They are but we now have to change the form action first. I did add a step to the website_crm tour we already had to reflect this change, and everything is working as it was before except for the fact we need to enable `Create an opportunity` action.

Reply:
Yes but that doesn't seems acceptable to me, the module is not doing what it is supposed to do, unless the enduser actually enter edit mode and change a specific option, which enduser will never, never, never do.

Reply:
IMO, on module install post init hook, we should at least write on the website.contactus page(s) to change `data-model_name` to `crm.lead` so the module is "ready to work" as expected and as it is without this task.

Reply:
What does not seem acceptable for @rdeodoo here was normally decided with @sbuhl and @JKE-be (as written in the spec)... did you discuss with him or did you end up implementing the post init hook stuff @tsm-odoo ? I don't think it is a viable idea, you don't know if there is even a form in this contact us page :grimacing: 

Original goal: contact us page is static so no magic involved in what it does: this is a pre-made form for you to contact the company -> you want it to do/show other things? Edit the page.

We can change our mind of course but let's not enter an infinite loop of spec changes please :wink: @sbuhl as this part of the spec is dubious since the beginning maybe it's time to involve FP so this spec does not change again? We already changed our mind about it like 3 times before ^^"

Reply:
@qsm-odoo No it's not implemented, indeed there were some issues like the fact that all form models have different field names... We discussed it during last week meeting and @JKE-be  told us that it was indeed the specs. Still, @rdeodoo  thinks it's a bad idea :
> I don't agree with the spec to remove the "ready to work" capacility of the website_crm module, I am pretty sure no one will understand they have to edit the form to change the action after installing the module.. But that's not up to me to decide..

Anyway, @rdeodoo told me he was going to last review this task today... Let me know if something changes :)

Reply:
Ok :wink: Yes, @JKE-be and I had the same fears as @rdeodoo but, specs :wink: I actually see the value in having the contact us page static... and keeping the CRM "auto" feature seems impossible/complicated functionally and technically so compromise... :grimacing: 

IMO this is the kind of stuff our navbar warning task could have been about too :( "Hint: your contact us page uses a form which sends mails, what about creating opportunities?" cc @sbuhl :wink: Useful hint, warning and error reports >>>>>> useless tours that nobody follows (especially when it's only for admins...)